README.md

```diff
@@ -41,7 +41,13 @@ This repository hosts samples for the [Windows App SDK](https://github.com/micro
 - [Windows Input and Composition Gallery](https://github.com/microsoft/WindowsCompositionSamples): This collection of samples showcases Microsoft.UI.Composition and Microsoft.UI.Input APIs.
 - [WinUI 3 Gallery](https://github.com/microsoft/WinUI-Gallery/): This is a sample app that showcases all of the WinUI 3 controls in action.
 
-## Requirements
+### Widgets
+- [Widgets](Samples/Widgets): These samples demonstrate how to author Windows Widgets with the Windows App SDK.
+
+### Windows AI Foundry
```

Comment:
Can we also move over the other AI samples to this new folder and also update them to reflect the changes.  Feel free to cherry pick that commit from my PR.

Other samples: https://github.com/microsoft/WindowsAppSDK-Samples/tree/feature/experimental-may2/Samples/WindowsCopilotRuntime

Relevant commits from my PR:

https://github.com/microsoft/WindowsAppSDK-Samples/pull/465/commits/9fa199ab617b7e67915ebe59071ea44a428da36f

https://github.com/microsoft/WindowsAppSDK-Samples/pull/465/commits/93d39deabe0e9e0ae01d27ddb67f783f9f4f47b3

Scott also had some minor feedback there.

Reply:
Out of scope of this PR. These samples's owners should make this change. that would better.